### PR TITLE
Modified methods to throw NOTFOUNDEXCEPTION instead RuntimeEror in ne…

### PR DESCRIPTION
**Pull request by aparna-cimpress**
…o4j_proxy (metadata_service)

### CheckList

Make sure you have checked **all** steps below to ensure a timely review.

- [ ] PR title addresses the issue accurately and concisely, including a title prefix.
- [ ] PR includes a summary of changes.
- [ ] My commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"

### CODE REVIEW

**Comment by aparna-cimpress** [comment]
Try block in _put_resource_description & put_column_description method in neo4j_proxy in metadata service throws RuntimeError when table or column doesn’t exist. I have changed it to throw NotFoundException with 404 status code.

**Review by dorianj — APPROVED** [review]

**Review by dorianj — APPROVED** [review]

**Comment by aparna-cimpress** [comment]
Hey, @dorianj  I think there is one more step of approval required. I am not sure, once approved then why need extra approval.

**Comment by aparna-cimpress** [comment]
Hey, @dorianj Can you please look at this PR once?? I think your approval is required again.

**Pull request merged**
